extensions/shared/external-media/constants.js

```diff
@@ -135,3 +135,36 @@ export const PEXELS_EXAMPLE_QUERIES = [
 	'abstract',
 	'sky',
 ];
+export const DATE_RANGE_ANY = 'ANY';
+export const DATE_RANGE_LAST_7_DAYS = 'LAST_7_DAYS';
+export const DATE_RANGE_LAST_30_DAYS = 'LAST_30_DAYS';
+export const DATE_RANGE_LAST_6_MONTHS = 'LAST_6_MONTHS';
+export const DATE_RANGE_OLDER = 'OLDER';
+export const DATE_RANGE_CUSTOM = 'CUSTOM';
+export const GOOGLE_PHOTOS_DATE_PRESETS = [
+	{
+		value: DATE_RANGE_ANY,
+		label: __( 'Any time', 'jetpack' ),
+	},
+	{
+		value: DATE_RANGE_LAST_7_DAYS,
+		label: __( 'Last 7 days', 'jetpack' ),
+	},
+	{
+		value: DATE_RANGE_LAST_30_DAYS,
+		label: __( 'Last 30 days', 'jetpack' ),
+	},
+	{
+		value: DATE_RANGE_LAST_6_MONTHS,
+		label: __( 'Last 6 months', 'jetpack' ),
+	},
+	{
+		value: DATE_RANGE_OLDER,
+		label: __( 'Older than a year', 'jetpack' ),
+	},
+	// TODO: Implement a UI for selecting month & year.
```

Comment:
Intentionally left in, as the filtering logic is fully implemented. UI will follow in a different PR.